src/test/unit_tests/test_s3_bucket_policy.js

```diff
@@ -192,6 +192,8 @@ mocha.describe('s3_bucket_policy', function() {
     });
 
     mocha.it('should be able to set bucket policy when none set', async function() {
+        const self = this; // eslint-disable-line no-invalid-this
```

Comment:
why we can not use this.timeout()? lint error?

Reply:
both ways cause lint errors. other tests in this file use ```const self = this``` so I did it this way to be consistent with the other tests